lib/_tls_wrap.js

```diff
@@ -1099,7 +1099,7 @@ exports.connect = function(...args /* [port,] [host,] [options,] [cb] */) {
   var cb = args[1];
 
   var defaults = {
-    rejectUnauthorized: '0' !== process.env.NODE_TLS_REJECT_UNAUTHORIZED,
+    rejectUnauthorized: process.env.NODE_TLS_REJECT_UNAUTHORIZED !== '0',
```

Comment:
Another example where I think it hurts legibility: the line is so long that the type check almost falls off the screen.

Reply:
Here the `process.env` part is actually something I consider less important to look at. So I have to move further to see that the comparison is with the `NODE_TLS_REJECT_UNAUTHORIZED` env.